tests/manage/mcg/ui/test_mcg_ui.py

```diff
@@ -331,11 +332,11 @@ def test_obc_creation_and_deletion(
 
         obc_ui_obj = ObjectBucketClaimsTab()
 
-        if (
-            config.DEPLOYMENT["external_mode"]
-            and storageclass == "ocs-storagecluster-ceph-rgw"
-        ):
-            storageclass = "ocs-external-storagecluster-ceph-rgw"
+        if storageclass == constants.DEFAULT_STORAGECLASS_RGW:
+            storageclass = get_default_storage_class_name(
+                constants.OCS_COMPONENTS_MAP["rgw"]
```

Comment:
should be `get_default_storage_class_name(constants.OCS_COMPONENTS_MAP["rgw"], external_mode=True)`